src/schema/auth.ts

```diff
@@ -0,0 +1,31 @@
+type AccessType = 'online' | 'offline';
+type Provider = 'google' | 'microsoft';
+
+export type AdminConsentAuth = SharedAuthParams & {
+  credentialId: string;
+};
+
+type SharedAuthParams = {
+  accessType?: AccessType;
+  prompt?: string;
+  redirectUri: string;
+  scope?: Array<string>;
+  includeGrantScopes: boolean;
```

Comment:
i don't see this in the UAS docs o.o

Reply:
It's still inside the codebase so maybe they forgot to add it to the docs 🤔